### PR TITLE
docs: rewrite security sections

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Rewrite security sections

## Context:

Follow-up PR after https://github.com/renovatebot/renovate/pull/14773

Copy and paste of the security section from our main Renovate readme. 

Now our security sections are the same in the main Renovate repository and our docs publishing repository.

### CODE REVIEW

File: .github/contributing.md
Comment (HonkingGoose) [inline .github/contributing.md:3]: I'm not sure if `docs:` or `chore:` is the best semantic commit scope.
I'll let you maintainers decide. 😉

Comment (viceice) [reply .github/contributing.md:3]: Either is fine, i would prefer docs

**Review by viceice — APPROVED** [review]

**Pull request merged**
